packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -90,8 +105,49 @@ const App: React.FC<AppProps> = observer(
       splitPaneRef,
     })
 
+    function onNavItemClick (index: number) {
+      if (activeIndex !== index) {
+        return setActiveIndex(index)
+      }
+
+      setActiveIndex(undefined)
+    }
+
+    const items: NavItem[] = [
+      {
+        id: 'file-explorer-nav',
+        title: 'File Explorer',
+        icon: 'copy',
+        interaction: {
+          type: 'js',
+          onClick: () => {
+            onNavItemClick(0)
+            setIsSpecsListOpen(!isSpecsListOpen)
+          },
+        },
+      },
+    ]
+
+    items.push(
+      {
+        id: 'docs-nav',
+        title: 'Cypress Documentation',
+        location: 'bottom',
+        icon: 'book',
+        interaction: {
+          type: 'anchor',
+          href: 'https://on.cypress.io/component-testing',
+        },
+      },
+    )
```

Comment:
Can we extract this in a separate file?

Reply:
Any benefit to moving this elsewhere? Seems like it belongs here - if we move it elsewhere, we then need to pass the `onClick` function (and any other click events we might add). Seems like un-necessary redirection. 🤔